jboss/container/wildfly/launch/tracing/added/launch/tracing.sh

```diff
@@ -37,13 +37,15 @@ function configureByCLI() {
       end-if
 EOF
     else
-      cat << 'EOF' >> ${CLI_SCRIPT_FILE}
-      if (outcome == success) of /subsystem=microprofile-opentracing-smallrye:read-resource
-        /subsystem=microprofile-opentracing-smallrye:remove()
-      end-if
-      if (outcome == success) of /extension=org.wildfly.extension.microprofile.opentracing-smallrye:read-resource
-        /extension=org.wildfly.extension.microprofile.config-smallrye:remove()
-      end-if
+      if [ "x${WILDFLY_TRACING_ENABLED}" == "xfalse" ]; then
```

Comment:
It's a really minor detail, we could use elif and remove the nested if/fi inside of the outer if

Reply:
Yeray, thank-you, fixed.